src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildACppCliProject.cs

```diff
@@ -24,21 +24,22 @@ public GivenThatWeWantToBuildACppCliProject(ITestOutputHelper log) : base(log)
         {
         }
 
-        [FullMSBuildOnlyFact(Skip = "https://github.com/dotnet/sdk/issues/11008")]
+        [FullMSBuildOnlyFact]
         public void It_builds_and_runs()
         {
             var testAsset = _testAssetsManager
                 .CopyTestAsset("NetCoreCsharpAppReferenceCppCliLib")
-                .WithSource();
+                .WithSource()
+                .WithProjectChanges((projectPath, project) => AddPackageReference(projectPath, project, "NewtonSoft.Json", "13.0.1"));
 
             // build projects separately with BuildProjectReferences=false to simulate VS build behavior
             new BuildCommand(testAsset, "NETCoreCppCliTest")
-                .Execute("-p:Platform=x64")
+                .Execute("-p:Platform=x64", "-p:EnableManagedpackageReferenceSupport=true")
```

Comment:
Instead of passing this on the command line, can we make it part of the project, either by including it directly in the test asset, or else in the `WithProjectChanges` call?

Same with other instances of this.